src/components/form/personalinformation/AddPersonalInformation.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable require-jsdoc */
+import * as Yup from 'yup';
+import PropTypes from 'prop-types';
+import { Formik, Form } from 'formik';
+import React, { useContext } from 'react';
+
+import Button from '~/components/button/Button';
+import * as storage from '~/storage/LocalStorage';
+import { FormContext } from '~/components/FormContext';
+import InputText from '~/components/inputtext/InputText';
+import FormHeader from '~/components/formheader/FormHeader';
+import * as personalInfoUtils from '~/utilities/objects/PersonalInformation';
+import OutsideClickDetector from '~/components/detector/OutsideClickDetector';
+
+const AddPersonalInformation = ({ onClose, isEdit }) => {
+  const { data } = useContext(FormContext);
+
+  const validatePersonalInformation = Yup.object().shape({
```

Comment:
shall we move this to /validation folder?

Reply:
moved